irr/src/COpenGLDriver.cpp

```diff
@@ -42,7 +42,7 @@ bool COpenGLDriver::initDriver()
 		return false;
 	ExposedData = ContextManager->getContext();
 	ContextManager->activateContext(ExposedData, false);
-	GL.LoadAllProcedures(ContextManager);
+	GL.LoadAllProcedures(ContextManager, Params.DriverType == video::EDT_OGLES2);
```

Comment:
this driver is GL-only
```suggestion
	GL.LoadAllProcedures(ContextManager, false);
```